Text advanture game/Text advanture game/main.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  main.swift
+//  Text advanture game
+//
+//  Created by Oscar Victoria Gonzalez  on 10/1/19.
+//  Copyright © 2019 Oscar Victoria Gonzalez . All rights reserved.
+//
+
+import Foundation
+
+let transportation1 = "4 train"
+let transportation2 = "2 train"
+
+let message = ("You've landed your first job interview as a software developer, you need to get there as soon as possible and your only form of transportation is the train. You have two choices. The \"4 train\" or the \"2 train\". Choose wisely.")
+
+print(message)
+
+
+
+let transportation  = readLine()!
+
+if transportation == transportation2 {
+    print("You chose to take the \(transportation2), however due to an accident halfway through your commute, your train stops and remains in the same spot for 2 hours. You dont make it to your job interview")
+}
+
+
+if transportation == transportation1 {
+ print("You've chosen to take the \(transportation1). Now you're on your way and suddently the train stops. The conductor says the train will skip your stop due to maintance issues. You have less than 30 minutes to get to your destinantion and the closest stop is 10 minutes away from your destination. You can choose to get off right now and try to get on an uber or you can stay and walk to your destination from the nearest stop. Will you stay or get off?")
+}
+
+
+    
+
+
+
+
+
+
+
+let choice = readLine()!
```

Comment:
Good job in making it a Let and not a var because you're not changing anything from the user's input